### PR TITLE
transfered gdocs writeup to markdown

### PR DESCRIPTION
**Pull request by northdpole**
this is a simple article we put together with @ThunderSon , it can serve as a basis for this project

### CODE REVIEW

File: writeups/owasp_in_sdlc/owasp_in_sdlc.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+The Requirements gathering process tries to answer the question: _"What is the system going to do?"_ At this stage, the [SAMM project](https://owaspsamm.org/model/) offers 3 distinct maturity levels covering both [in-house](https://owaspsamm.org/model/design/security-requirements/stream-a/) software development and [third party](https://owaspsamm.org/model/design/security-requirements/stream-b/) supplier security.
+
+![SAMM Requirements](images/OWASP-in0.png "SAMM Requirements")
+
+Organisations can use these to add solid security considerations at the start of the Software Development or Procurement process.
+
+These general security considerations can be audited by using a subsection of the ASVS controls in section V1 as a questionnaire. This process attempts to ensure that every feature has concrete security considerations.
+
+In case of internal development and if the organisation maps Features to Epics, the [Security Knowledge Framework](https://securityknowledgeframework.org/) can be used to facilitate this process by leveraging its questionnaire function, shown below.
```
Comment (ThunderSon) [inline writeups/owasp_in_sdlc/owasp_in_sdlc.md:25]: This is missing a picture, no?

File: writeups/owasp_in_sdlc/owasp_in_sdlc.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+**Example low maturity scenario:**
+
+Customer commission's application that aggregates data from a number of stores offering a unified API for all data querying and retrieval. The users of the system will be customer employees. The application needs to be accessible from the internet.
```
Comment (ThunderSon) [inline writeups/owasp_in_sdlc/owasp_in_sdlc.md]: This sentence was a bit hard to read. Could we maybe improve it?

**Review by ThunderSon — CHANGES_REQUESTED** [review]
After validating the missing image, I will add numbering on the figures to make it easier to reference. We'll be good to send it out as a first release for other's reviews.

File: writeups/owasp_in_sdlc/owasp_in_sdlc.md (hunk 1 above)
Comment (northdpole) [reply writeups/owasp_in_sdlc/owasp_in_sdlc.md:25]: it's a link

**Review by ThunderSon — APPROVED** [review]
I'd say the first picture can be cut shorter, and the last one. It is good to be merged though.

**Pull request merged**